pkg/dt/fdt.go

```diff
@@ -411,3 +413,35 @@ func (fdt *FDT) Write(f io.Writer) (int, error) {
 	_, err := w.Write(strs)
 	return w.N, err
 }
+
+// NodeByName finds a node by name.
+func (fdt *FDT) NodeByName(name string) (*Node, bool) {
+	return fdt.RootNode.Find(func(n *Node) bool {
+		return n.Name == name
+	})
+}
+
+// Bytes returns a []byte for the "data" property of a node,
+// or an error if the node has no data property.
+func (fdt *FDT) Bytes(name string) ([]byte, error) {
```

Comment:
It looks like this is specific to FIT, but not Device Tree in general. Perhaps make it a utility in a separate package called `fit` or rename to `FitBytes`.

Reply:
done

Reply:
not really. IIRC data is a comment property for a node?